c2corg_ui/static/partials/feedtable.html

```diff
@@ -0,0 +1,66 @@
+<div ng-show="wfeedCtrl.documents.length > 0"
+  infinite-scroll="wfeedCtrl.getDocumentsFromWhatsnewFeed()"
+  infinite-scroll-immediate-check="false"
+  infinite-scroll-distance="0.7"
+  infinite-scroll-disabled="wfeedCtrl.busy || wfeedCtrl.error || wfeedCtrl.feedEnd || wfeedCtrl.noFeed">
+
+  <div class="table-responsive">
+    <table class="text-center feedtable table table-condensed table-striped table-hover">
+
+      <p class="text-success text-center" ng-if="wfeedCtrl.userId"
+         ng-init="user = wfeedCtrl.userId;" ng-model="amount = wfeedCtrl.feedLength;">
+        This feed for the user {{user}} contains {{amount}} changes.
+      </p>
+      <p class="text-success text-center" ng-if="!wfeedCtrl.userId"
+         ng-model="amount = wfeedCtrl.feedLength;">
+        This feed contains {{amount}} changes.
+      </p>
+
+      <tr>
+        <th class="text-center" translate>title</th>
+        <th class="text-center" translate>type</th>
+        <th class="text-center" translate>written_at</th>
+        <th class="text-center" translate>quality</th>
+        <th class="text-center" translate>username</th>
+        <th class="text-center" translate>comment</th>
+      </tr>
+
+      <tr class="feed-list"
+              ng-repeat="doc in wfeedCtrl.documents track by doc.version_id"
+          ng-init="type = wfeedCtrl.documentType(doc.document.type); console.log(type);" >
+
+        <td class="text-left">
+          <a href="{{wfeedCtrl.getDocHref(doc)}}">{{::doc['document']['title']}}</a> ({{::doc['lang']}})
+          <span ng-if="doc['document']['version'] > 1 && (doc['document']['type'] !== 'i' && doc['document']['type'] !== 'o')">
```

Comment:
I wasn't sure how we store old versions of these documents because many created URL addresses of these document types weren't accessible. I simply hide them but it is possible that it was caused by some other problem. For example 1st version of this image can't be opened (probably created as private and later changed to collaborative?) https://www.camptocamp.org/images/history/851612/fr